src/main/java/application/onlinebookstore/controller/BookController.java

```diff
@@ -19,49 +22,67 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/api/books")
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
+    @Operation(summary = "Get all books",
+            description = "Get a list of all available books "
+                    + "or you can set pageable ?page=?&size=?")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.getAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Get book by id",
+            description = "Get a book in repository by id")

```

Comment:
Not in repository. You get book from db. delete and update also in db. Fix it for this controller
```suggestion
            description = "Get book from DB by id")

```